modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -53,8 +82,55 @@ No Premium Plugins released with {productname} 6.0 are backwards compatible with
 
 The following premium plugin updates were released alongside {productname} 6.0.
 
+=== Accessibility Checker
+
+The following change was made to the xref:a11ychecker.adoc[Accessibility Checker] plugin:
+
+The default value for `a11ychecker_html_version` was `html4`.
+
+It is now `html5`.
+
+
+=== Advanced Tables
+
+The following changes and fixes were made to the xref:advtable.adoc[Advanced Tables] plugin:
+
+With all table logic, except table UI components, now part of xref:6.0-release-notes-core-changes.adoc#changed-plugins-table[core {productname} editor], the `advtable` plugin now only requires the `table` plugin be enabled to use the Advanced Table-specific toolbar buttons and menu items.
+
+Previously, the `getRowType()` method of the series generator info argument incorrectly returned `header` when a `tfoot` row contained only header cells.
+
+With this update, `getRowType()` correctly returns `footer` when a `tfoot` row contains only header cells.
+
+Previously tables containing a `colgroup` did not sort correctly. As of this release, this error has been corrected: tables with a `colgroup` now sort correctly, as expected.
+
+
+=== Comments
+
+The following changes and fixes were applied to the xref:introduction-to-tiny-comments.adoc[Comments] plugin:
+
+Because of how a comment’s active state was captured when the comment was made active, this active state was not excluded when capturing editor content from a comment.
+
+With this release, a comment’s active state is captured differently and, consequently, this state is excluded when comment content is captured.
+
+Previously, every time the insertion point was placed within an extant comment and then placed anywhere outside of the comment body, an undo level was added. That is, every time focus was switched either to or from a content, an undo level was added.
+
+With this update, shifting focus to or from a comment no longer adds undo levels, as expected.
+
+Previously, the Comments plugin’s scroll position logic lead to, in some edge cases, the last comment not scrolling into view correctly.
+
+The logic has been corrected and, with this update, all comments scroll into view as expected.
+
+Previously, when a Comment was added to a {productname} instance, the data in `tinycomments` was automatically, and incorrectly, added to the *Find and Replace* Find text-entry input field.
```

Comment:
This isn't completely correct... it was incorrectly added to `editor.selection.getContent()` which meant it showed in dialogs, such as Find and Replace and Insert Link.